src/compiler/scala/tools/nsc/backend/jvm/opt/ClosureOptimizer.scala

```diff
@@ -325,7 +325,7 @@ class ClosureOptimizer[BT <: BTypes](val btypes: BT) {
         insns.insertBefore(invocation, new InsnNode(DUP))
         INVOKESPECIAL
     }
-    val isInterface = bodyOpcode == INVOKEINTERFACE
+    val isInterface = classBTypeFromInternalName(lambdaBodyHandle.getOwner).isInterface.get
```

Comment:
Could we ever hit a case where we don't `classBTypeFromInternalName.apply` or `.get` fails, say if the lambda target class was not in the current compilation classpath? (Maybe a lambda had previously been inlined into the class from which we are inlining.)